src/App.js

```diff
@@ -1,8 +1,26 @@
 import React from 'react'
-import Burger from './Burger'
+import Burger from './components/Burger/Burger'
+import Kebab from './components/Kebab/Kebab'
+import Minced from './components/Minced/Minced'
+import Nuggets from './components/Nuggets/Nuggets'
+import Sausages from './components/Sausages/Sausages'
+import styled from 'styled-components'
 
 export default function App() {
     return(
+        <>
+        <HeadlineStyled>All vegan products</HeadlineStyled>
         <Burger />
```

Comment:
It seems like the components Burger, Kebab, Minced and so on are very similar. Maybe you can make a shared component and just enter the category as a prop?